smu/geometry/bond_length_distribution.py

```diff
@@ -232,6 +250,7 @@ def from_sparse_dataframe(cls, df_input, right_tail_mass, sig_digits):
           set(df_input['length_str']).difference(df_lengths['length_str'])))
 
     df['count'].fillna(0, inplace=True)
+    df['count'] = interpolate_zeros(np.array(df['count']))
```

Comment:
You don't need this both here and above.

Reply:
Agree...